static/js/core.js

```diff
@@ -1,226 +1,89 @@
-						src = src + '?fmt=png';
-					} else {
-						// Old asset manager and locally assets
-						src = src.match(/.*\/(.+?)\./)[0] + 'png';
-					}
-					node.setAttribute("src", src);
-				});
-			}
-		}
-	};
-  // BAD, USE CSS
-    core.resourceHubBoxes = function () {
-        Y.all(".resource").on('click', function (e) {
-            e.preventDefault();
-            e.stopPropagation();
-            if (e.currentTarget.one('a') !== null) {
-                window.location = e.currentTarget.one('a').get("href");
-            }
-        });
-    };
-    // NEED, WRITE IN ES5
-    core.mobileNav = function () {
-        if (Y.one('.header-search')) {
-            Y.one('.nav-primary').insert('<a class="search-toggle"></a><a id="menu" class="nav-toggle">menu</a>', 'before');
-        } else {
-            Y.one('.nav-primary').insert('</a><a id="menu" class="nav-toggle">menu</a>', 'before');
-        }
+    if (navPrimary) {
+        // Add the navigation toggle
+        navToggle = document.createElement('a');
+        navToggle.classList.add('nav-toggle');
+        navToggle.id = 'menu';
+        navToggle.innerHTML = 'menu';
+        header.insertBefore(navToggle, navPrimary);
 
-        Y.all('.nav-toggle').on('click', function (e) {
+        // Adds a click listener to the navigation toggle
+        navToggle.addEventListener('click', function(e) {
             e.preventDefault();
-            if (Y.one('.header-search')) {
-                Y.one('.header-search').removeClass('active');
+            // Hide the search field
+            if (headerSearch) {
+                headerSearch.classList.remove('active');
             }
-
-            Y.all('.nav-toggle').toggleClass('active');
-            Y.all('header nav ul').toggleClass('active');
-            Y.all('.nav-primary').toggleClass('active');
+            // Toggle the navigations display state
+            navToggle.classList.toggle('active');
+            navPrimary.classList.toggle('active');
+            document.querySelector('#nav ul').classList.toggle('active');
         });
+    }
 
-        if (Y.one('.header-search')) {
-            Y.one('.search-toggle').on('click', function (e) {
-                e.preventDefault();
-
-                Y.all('.nav-toggle').removeClass('active');
-                Y.all('header nav ul').removeClass('active');
-                Y.all('.nav-primary').removeClass('active');
-
-                Y.one('.header-search').toggleClass('active');
-                Y.one('.header-search .form-text').focus();
-            });
-        }
-
-        if (Y.one('.breadcrumb li a')) {
-            Y.one('.breadcrumb > li a').insert('<span class="after"></span>');
-            Y.one('.breadcrumb li a .after').on('click', function (e) {
-                e.preventDefault();
-                Y.one('.nav-secondary').toggleClass('open');
-                core.setLocalStorage('onboard');
-            });
-        }
-
-    };
-    // REWRITE IN ES6
```

Comment:
No, I dont think we need to worry about persisting the onboarding of the mobile nav as its a very common pattern.